HousingRepairsOnlineApi/Properties/launchSettings.json

```diff
@@ -28,8 +28,14 @@
         "AUTHENTICATION_IDENTIFIER": "<REPLACE WITH A UNIQUE AUTHENTICATION IDENTIFIER>",
         "JWT_SECRET" : "<REPLACE WITH A SECRET>",
         "ADDRESSES_API_URL" : "https://<REPLACE WITH ADDRESSES API URL>",
-        "SCHEDULING_API_URL" : "https://<REPLACE WITH SCHEDULING API URL>"
+        "SCHEDULING_API_URL" : "https://<REPLACE WITH SCHEDULING API URL>",
+        "COSMOS_ENDPOINT_URL": "https://<REPLACE WITH COSMOS ENDPOINT ULR>",
+        "COSMOS_AUTHORIZATION_KEY": "REPLACE WITH COSMOS AUTH KEY",

```

Comment:
Please enclose configuration values that should be replaced within angle brackets for consistency.